code/game/gamemodes/dynamic/dynamic.dm

```diff
@@ -609,14 +609,16 @@ GLOBAL_VAR_INIT(dynamic_forced_threat_level, -1)
 /// Mainly here to facilitate delayed rulesets. All roundstart rulesets are executed with a timered callback to this proc.
 /datum/game_mode/dynamic/proc/execute_roundstart_rule(sent_rule)
 	var/datum/dynamic_ruleset/rule = sent_rule
-	if(rule.execute())
+	var/execute_result = rule.execute()
+	if(execute_result && execute_result != NOT_ENOUGH_PLAYERS)
```

Comment:
```suggestion
	if(execute_result != NOT_ENOUGH_PLAYERS)
```
null != value

Reply:
null/0 is not NOT_ENOUGH_PLAYERS, which means this condition would pass. I do not want it to pass on null/0